jcloud/parsers/__init__.py

```diff
@@ -11,7 +11,7 @@ def get_main_parser():
     from .remove import set_remove_parser
 
     # create the top-level parser
-    parser = set_base_parser()
+    parser = set_base_parser(parser=parser)
 
     sp = parser.add_subparsers(
         dest='cli',
```

Comment:
this also has to be changed, say `jc_cli`

Reply:
alright

Reply:
done